.eslintrc.yml

```diff
@@ -68,9 +68,6 @@ rules:
   no-unreachable: error
   no-undef: error
 
-  # These are disabled for now and we will slowly turn them on
-  require-await: off
```

Comment:
The ["When not to use it"](https://eslint.org/docs/rules/require-await#when-not-to-use-it) section seems to apply here!